test/e2e/far_e2e_test.go

```diff
@@ -134,27 +135,28 @@ var _ = Describe("FAR E2e", func() {
 		When("running FAR to reboot two nodes", func() {
 			It("should successfully remediate the first node", func() {
 				checkRemediation(nodeName, nodeBootTimeBefore, pod)
-				remediationTimes = append(remediationTimes, time.Since(startTime))
+				remediationsTime = append(remediationsTime, time.Since(startTime))
```

Comment:
hm, I think the old name was correct... 🤔 
multiple time*s* of a single remediation
this now sounds like overall time of all remediation*s*